fast_rnnt/python/fast_rnnt/mutual_information.py

```diff
@@ -285,9 +285,10 @@ def mutual_information_recursion(
         for s_begin, t_begin, s_end, t_end in boundary.tolist():
             assert 0 <= s_begin <= s_end <= S
             assert 0 <= t_begin <= t_end <= T
-    # The following assertions are for efficiency
-    assert px.is_contiguous()
-    assert py.is_contiguous()
+
+    # The following statements are for efficiency
+    px, py = px.is_contiguous(), py.is_contiguous()
```

Comment:
I think you meant:
```
px, py = px.contiguous(), py.contiguous()
```

Reply:
Yes, Thank! I think I had fixed it in the consecutive commit.